### PR TITLE
chore: Add GitHub actions (#283)

### LINKED ISSUE #283: Switch to Github Actions
To remove the amount of external dependencies (here travis), I'd like to try Github Actions for this project.

It'll be simple: just create 2 worksflows, first one run `yarn lint`, second one `yarn test`

### PR DESCRIPTION
**Pull request by sathia27**
fixes #283 

Current steps:
1. Checkout
2. yarn install and lint
3. yarn test

### CODE REVIEW

**Comment by amaurymartiny** [comment]
Thanks! Hmm, I am not sure why the actions are not appearing in the checks. Maybe I should activate Actions first

**Comment by amaurymartiny** [comment]
Hey @sathia27, could you instead replace the existing main.yml (which I just created). I believe after that the Actions will appear in the checks. Thanks a lot!

Two additional notes:
- Could you create 2 separate jobs: one `lint` and one `test` (instead of running both in the same job)
- Could you delete .travis.yml?

Thanks!

**Comment by sathia27** [comment]
>     * Github action badge

Sure will work on it.

**Comment by sathia27** [comment]
@amaurymartiny 
1. Kept it as seprate job
2. removed travis.yml
3 added github badge in README

Please check and let me know if something needs to be changed.

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+  setup:
```
Comment (amaurymartiny) [inline .github/workflows/main.yml]:
```suggestion
  lint:
```

**Comment by sathia27** [comment]
> Thanks, this looks good! Just a small rename nit in your PR.
> 
> Could you fix the conflict?

**Pull request merged**